### PR TITLE
test: clean up TextArea integration tests

### PR DESCRIPTION
**Pull request by vursen**
## Description

The PR cleans up TextArea integration tests.

# Type of change

- [x] Internal

### CODE REVIEW

File: vaadin-text-field-flow-parent/vaadin-text-field-flow-integration-tests/src/test/java/com/vaadin/flow/component/textfield/tests/TextAreaPatternIT.java [hunk 1]
```diff
@@ -1,129 +0,0 @@
...
-        setNumberPattern.click();
-        textArea.sendKeys("1234");
-        blur();
-
-        assertClientSideValid();
-        assertServerSideValid();
-    }
-
-    @Test
-    public void validPattern_invalidInput_isInvalid() {
```
Comment (vursen) [inline vaadin-text-field-flow-parent/vaadin-text-field-flow-integration-tests/src/test/java/com/vaadin/flow/component/textfield/tests/TextAreaPatternIT.java:87]: **note:** Already covered by this test:

https://github.com/vaadin/flow-components/blob/4477ce194a7491f8e7777f40f3e0162d421d2d90/vaadin-text-field-flow-parent/vaadin-text-field-flow-integration-tests/src/test/java/com/vaadin/flow/component/textfield/tests/validation/TextAreaBasicValidationIT.java#L120-L125

File: vaadin-text-field-flow-parent/vaadin-text-field-flow-integration-tests/src/test/java/com/vaadin/flow/component/textfield/tests/TextAreaPatternIT.java [hunk 2]
```diff
@@ -1,129 +0,0 @@
...
-
-    @Test
-    public void validPattern_validInput_isValid() {
-        setNumberPattern.click();
-        textArea.sendKeys("1234");
-        blur();
-
-        assertClientSideValid();
-        assertServerSideValid();
-    }
```
Comment (vursen) [inline vaadin-text-field-flow-parent/vaadin-text-field-flow-integration-tests/src/test/java/com/vaadin/flow/component/textfield/tests/TextAreaPatternIT.java:84]: **note:** Already covered by this tests:

https://github.com/vaadin/flow-components/blob/4477ce194a7491f8e7777f40f3e0162d421d2d90/vaadin-text-field-flow-parent/vaadin-text-field-flow-integration-tests/src/test/java/com/vaadin/flow/component/textfield/tests/validation/TextAreaBasicValidationIT.java#L120-L130

File: vaadin-text-field-flow-parent/vaadin-text-field-flow-integration-tests/src/test/java/com/vaadin/flow/component/textfield/tests/TextAreaPatternIT.java [hunk 3]
```diff
@@ -1,129 +0,0 @@
...
-        setNumberPattern.click();
-        textArea.sendKeys("abcd");
-        blur();
-
-        assertClientSideInvalid();
-        assertServerSideInvalid();
-    }
-
-    @Test
-    public void validPattern_allowedCharPattern_invalidInput_inputShouldBePrevented() {
```
Comment (vursen) [inline vaadin-text-field-flow-parent/vaadin-text-field-flow-integration-tests/src/test/java/com/vaadin/flow/component/textfield/tests/TextAreaPatternIT.java:97]: **note:** I think this test doesn't make sense. Essentially, it makes sure that enabling `allowedCharPattern` really prevents input. That is already tested on the side of the web component, I believe.

File: vaadin-text-field-flow-parent/vaadin-text-field-flow-integration-tests/src/test/java/com/vaadin/flow/component/textfield/tests/TextAreaPatternIT.java [hunk 4]
```diff
@@ -1,129 +0,0 @@
...
-        textArea.sendKeys("abcd1234");
-        blur();
-
-        assertClientSideValid();
-        assertServerSideValid();
-        Assert.assertEquals("1234", currentValue.getText());
-    }
-
-    @Test
-    public void invalidPattern_anyInput_isValid() {
```
Comment (vursen) [inline vaadin-text-field-flow-parent/vaadin-text-field-flow-integration-tests/src/test/java/com/vaadin/flow/component/textfield/tests/TextAreaPatternIT.java:109]: **note:** This test doesn't quite make sense as setting an invalid pattern will throw a `PatternSyntaxException` anyway.

**Review by web-padawan — APPROVED** [review]

**Pull request merged**